resources/zib/zib-Pregnancy-EstimatedDateOfDelivery.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-Pregnancy-EstimatedDateOfDelivery" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-Pregnancy-EstimatedDateOfDelivery" />
+  <name value="ZibPregnancyEstimatedDateOfDelivery" />
```

Comment:
Ik mis in dit profiel nog de pattern op het .code element om aan te geven dat het over a terme datum gaat: [161714006](https://terminologie.nictiz.nl/art-decor/snomed-ct?conceptId=161714006) Estimated date of delivery

Reply:
Pattern code toegevoegd